content/en/docs/pipelines/tutorials/benchmark-examples.md

```diff
@@ -0,0 +1,184 @@
++++
+title = "Using the Kubeflow Pipelines Benchmark Scripts"
+description = "How to use the Kubeflow Pipelines Benchmark Scripts"
+weight = 10
++++
+
+This guide explains the Kubeflow Pipelines [benchmark scripts](https://github.com/kubeflow/pipelines/tree/master/tools/benchmarks)
+and demonstrates how to use them to collect basic performance data of a given
+Kubeflow Pipelines deployemnt.
+
+## Background and Overview
+
+The Kubeflow Pipelines benchmark scripts simulate typical workloads, apply them
+to the Kubeflow Pipelines deployment, and record the server latencies and
+pipeline run durations under the workloads.
+
+To simulate a typical workload, the benchmark script will use a pipeline
+manifest file, upload it to the Kubeflow Pipelines deployment as a pipeline or
+a pipeline version, and then run the uploaded pipeline or pipeline version
+multiple times simulatenuously. Users can specify the pipelines manifest used in
+the benchmark script. E.g., the preloaded samples in Kubeflow pipelines can be
+used. Moreover, it is also a good practice to use a representative pipeline
+manifest in light of the particular Kubeflow Pipelines use case. E.g., if a
+Kubeflow Piplines deployment is mainly used for pipelines of image recognition
+tasks, then it would be desirable to use an image recognition pipeline in the
+benchmark scripts.
+
+When a proper pipeline is chosen, the benchmark scripts will run it multiple
+times simultaneously on a given Kubeflow Pipelines deployment, and collects the
+server latencies and run durations. Among all the operations that the Kubeflow
+Pipelines can perform, running a pipeline is arguably the most unpredictable and
+costly one. Other operations, e.g., creating a pipeline (version) or creating an
+experiment, usually induce a predicatable and moderate cost. E.g., creating a
+pipeline version will introduce a new row in the pipeline versions table, a new
+file in minio server. The new file's size depends on the pipeline version's
+manifest. If we exclude the rare case of an extremely large manifest and assume
+an average sized manifest for each created pipeline version, the total cost of
+creating a pipeline version grows linearly with the number of pipeline versions.
+However, on the other hand, the cost of running a pipeline or a pipeline version
+involves much more uncertainty and sometimes quite high a cost. A pipeline or a
+pipeline version can have arbitrary components and hence running a pipeline or a
+pipeline version can incur arbitrary time and space complexities. E.g., the step
+in a pipeline can use a customized container imange which performs a super
+expensive training task. In addition, the runs in a Kubeflow Pipelines deployment
+also consume more DB space than pipelines, pipeline versions, experiments, etc.
+
+Therefore, when the performance and scalibility of Kubeflow Pipelines are in
```

Comment:
I changed it to "Therefore, in order to understand the performance and scalability pain points in a Kubeflow Pipelines instance, it is more effective to focus on the run operation
in workloads." My intention is to emphasize the "run operations" here, since run operation is the one  that can actually load test our KFP instance.